paper_sections/02-gams.Rmd

```diff
@@ -1,33 +1,39 @@
 
-#II: and introduction to Generalized Additive Models
-
-One of the most common model formulations in statistics is the generalized linear model [@McCullagh:1989ti] --- models that relate some response ($y$) to linear combinations of explanatory variables. We may allow allow the response to be distributed according to some exponential family distribution (e.g., letting the response be a trial, a count or a strictly positive number -- binomial, Poisson or Gamma distributions, respectively). The generalized additive modelling (GAM) framework [@Hastie:1990vg; @Ruppert:2003uc; @Wood:2006vg] allows the relationships between the explanatory variables (henceforth covariates) and the response to be described by smooth terms (usually *splines* [@deBoor:1978wq], but potentially other structures). In general we are then talking about models of the form:
+#II: an introduction to Generalized Additive Models
 
+One of the most common model formulations in statistics is the generalized linear model [@McCullagh:1989ti] that relates a response ($y$) to a linear combination of explanatory variables. The response is assumed to be conditionally distributed according to a member of the exponential family (e.g., letting the response be a trial, a count or a strictly positive real number --- binomial, Poisson or Gamma distributions, respectively). The generalized additive model [GAM; @Hastie:1990vg; @Ruppert:2003uc; @Wood:2006vg] allows the relationships between the explanatory variables (henceforth covariates) and the response to be described by smooth terms (usually *splines* [@deBoor:1978wq], but potentially other structures). In general we have models of the form:
 $$
-\mathbb{E}\left( y \right) = g^{-1}\left( \beta_0 + \sum_{j=1}^J f_j(x_j) \right),
+\mathbb{E}\left( y \right) = g^{-1}\left( \beta_0 + \sum_{j=1}^J f_j(x_j) \right)\,,
 $$
+where $\mathbb{E}(y)$ is the expected value of the response $y$ (with an appropriate distribution and link function $g$), $f_j$ is a smooth function of the covariate $x_j$, $\beta_0$ is an intercept term and $g^{-1}$ is the inverse link function. Here there are $J$ smooths and each is a function of only one covariate, though it is possible to construct smooths of multiple variables.
```

Comment:
\mathbb{E}(y) should be \mathbb{E}(Y) where Y~EF(\mu, \phi) (we want the expectation of the random variable Y, not the observation y) or somesuch, try to stick to the format at the start of chapter 4 of Simon's book?

Reply:
That makes sense to me. We should try to be as consistent as possible with Simon's notation.

Reply:
Agreed - I'm a geographer, masquerading as an ecologist, wielding math notation, so apologies for any abuses.

Reply:
No worries, just want to make sure things are correct :)